profile/README.md

```diff
@@ -0,0 +1,40 @@
+![Unleash_GitHub_Banner](https://user-images.githubusercontent.com/18158596/196736157-b20a5fbf-74ac-47cd-a991-fe6e2d2bc960.png)
+
+<p align="center">
+<b>The Open Source Feature Flagging Service</b>
+</p>
+
+![Unleash_GitHub_Mocks (2)](https://user-images.githubusercontent.com/18158596/196740001-569c6951-e989-4f7e-b31b-def5c40fa2aa.png)
+
+
+<!-- <img width="600" alt="Unleash Logo" src="https://user-images.githubusercontent.com/18158596/196730085-d168ff9e-8b1a-4790-8219-882336df6479.png"> -->
+
+**Unleash started as a side project.** We wanted to fill an essential space in the development stack and control releases, and, in the beginning, we were just a few contributors. Together we built a foundation for the amazing community that you see today. 🫂
+
+We love that our community is able to work with our product with all the freedom you could ever want. When you can just test, experiment, and release confidently, you'll end up with a well-rounded product with a ton of awesome interpretations. We're proud of you guys. 🙏
+
+That's enough about us. Have fun. Be kind. Let's **Unleash** great software together. 💪
+
+> [📈 See our roadmap →](https://github.com/orgs/Unleash/projects/5)
+
+## 🚀 Learn about Unleash
+Create with Freedom, Release With Confidence.
+
+[Main Open Source Repo](https://github.com/Unleash/unleash) - Unleash's Open Source codebase. 🖥️ </br>
+[Docker Hub](https://hub.docker.com/r/unleashorg/unleash-server) - Unleash containerized, just for you. 🎛️ </br>
+[Unleash Docs](https://docs.getunleash.io) - Dive deeper into the docs. 📚 </br>
+[How-to Guides](https://docs.getunleash.io/how-to) - All the how-to's answered for you. 🙋 </br>
+[Open Live Demo](https://www.getunleash.io/interactive-demo) - Unleash Official demo application using Unleash + Next.js.</br>
```

Comment:
Wait, does this use Next.js? I wasn't aware of that 😅

Reply:
I believe it does, it's a small detail we can update it later.